README.md

```diff
@@ -3,4 +3,66 @@
 
 # releng-reusable-workflows
 
-Reusuable workflows developed by LF Release Engineering
+GitHub actions/workflows developed by the Linux Foundation Release Engineering
+team.
+
+## Naming Conventions
+
+### File and Directory Names
+
+## Location, Naming and Labelling of Actions
+
+You can find actions in folders under the directory:
+
+    .github/actions/
+
+These can be of three basic types, as described in the [GitHub documentation]
+(<https://docs.github.com/en/actions/sharing-automations/creating-actions/about-custom-actions>)
+
+The table below describes the labelling and locations:
+
+| Description | Directory Suffix |
+| ----------- | ---------------- |
+| Composite   | -action          |
+| Javascript  | -javascript      |
+| Docker      | -docker          |
+
+Documentation takes the form of a README.md or README.rst file hosted alongside
+the YAML file in one of the directory locations described above.
+
+## Naming of Workflow YAML files
+
+The GitHub actions and workflows in this repository conform to the
+following naming conventions:
+
+| Filename Prefix | Functional Description                        |
+| --------------- | --------------------------------------------- |
+| call-           | Top-level parent workflows that call others   |
+| reuse-          | Reusable workflows implementing workflow_call |
+
+## Name/Labelling within Workflows
+
+Labelling must avoid consuming excessive space in the web/portal sidebar.
+With this in mind, some standard abbreviations help minimise text overflow.
+
+### Re-usable Workflows
+
+Reusable workflows are internally named with the prefix "[R]", followed by the
+workflow filename stripped of the "reuse-" prefix and ".yaml" suffix. An
+example of this in the workflow YAML file would be:
+
+`name: "[R] sonatype-lifecycle"`
+
+### Gerrit Workflows
+
+Gerrit 'calling' workflows may have different trigger event types. Those
+embedded behaviours mean a naming prefix helps distinguish the different
+types.
+
+They fall into three specific types:
+
+| Workflow Prefix | Description      | Filename in Repository |
+| --------------- | ---------------- | ---------------------- |
+| [GV]            | Gerrit (Verify)  | gerrit-verify.yaml     |
+| [GM]            | Gerrit (Merge)   | gerrit-merge.yaml      |
+| [G]             | Gerrit (Generic) | [either of the above]  |
```

Comment:
The `Generic` really should be listing a filename that is `gerrit-<something>`.yaml as we do technically actually support other types, just nobody is doing so in practice right now. To be specific this would be used for custom magic comment keyword triggers.